conans/model/editable_cpp_info.py

```diff
@@ -0,0 +1,82 @@
+# coding=utf-8
+import configparser
+import ntpath
+import os
+import posixpath
+from collections import defaultdict
+
+from conans.client.tools.files import load
+
+
+class EditableCppInfo(object):
+    WILDCARD = "*"
```

Comment:
`This is slightly different to other places in conan where we use fnmatch pattern matching. This implementation allows ``*:includedirs``, but not ``Project*:includedirs``, which is inconsistent with other conan use cases (search, build-modes, upload...)